packages/ra-ui-materialui/src/detail/ShowView.tsx

```diff
@@ -55,7 +65,9 @@ export const ShowView = (props: ShowViewProps) => {
     );
 };
 
-export type ShowViewProps = ShowProps;
+export type ShowViewProps<RecordType extends RaRecord = any> = ShowProps<
+    RecordType
+>;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why did you need that change?